scripts/provision.sh

```diff
@@ -25,8 +25,7 @@ init_datastore() {
   mkdir -p /hab/user/builder-datastore/config
   cat <<-EOT >/hab/user/builder-datastore/config/user.toml
 		max_locks_per_transaction = 128
-		dynamic_shared_memory_type = 'none'
```

Comment:
what would happen if this was missing and a customer was running the older postgres?

Reply:
It would likely use the POSIX option as I read the documentation.  The none option was removed in PostgreSQL 12 which first release October 3, 2019 and EOL'd November 21, 2024. I don't think I'm too worried about that because by the time of the removal the postgres team felt that shared mem support was reliable and pervasive enough that they removed and even the old server given me fits at the moment is Ubuntu 20.04.6 LTS (Focal Fossa) which was released after the postgres team's decision based on their discernment regarding shared mem support.